setup/symfony_server.rst

```diff
@@ -284,16 +284,46 @@ server provides a ``run`` command to wrap them as follows:
     # stop the web server (and all the associated commands) when you are finished
     $ symfony server:stop
 
-Configuring Workers
--------------------
+Configuration file
+------------------
 
 .. caution::
 
     This feature is experimental and could change or be removed at any time
     without prior notice.
 
+There are several options that you can set using a ``.symfony.local.yaml`` config file:
+
+.. code-block:: yaml
+
+    # Sets domain1.wip and domain2.wip for the current project
+    proxy:
+        domains:
+            - domain1
+            - domain2
```

Comment:
I'm wondering if this should not be moved to the proxy section 🤔.
Basically pin every part of the YAML to its dedicated feature section.

Reply:
I'm open to move it if it's really wanted?

Reply:
After thinking about this, I think it's better for now to keep all options of the config file in the same section, because it's not very long. In any case, if we realize that this is not working well, we can quickly update this in a future PR to do what @tucksaun suggested.